documentation/book/assembly-deployment-configuration-kafka.adoc

```diff
@@ -69,6 +69,10 @@ include::proc-manual-delete-pod-pvc-kafka.adoc[leveloffset=+1]
 
 include::proc-manual-delete-pod-pvc-zookeeper.adoc[leveloffset=+1]
 
+include::proc-renewing-ca-certs-manually.adoc[leveloffset=+1]
```

Comment:
These annotations can be used only within the maintenance window (when enabled). Would it therefore make sense to place the procedures only after the maintenance window assembly?

Reply:
I didn't realise that. Yes, I think it makes sense to place it after the maintenance time windows section. I'll add a note to both procedures stating that the maintenance time window must be active/enabled in order to manually renew/replace certs/keys (or words to that effect).

Reply:
Please can you clarify: Can I only apply the annotation within a maintenance time window?

Is it correct to state the following:

**Note:** You are only allowed to replace private keys within a maintenance time window.

Reply:
@laidan6000 No, you can set the annotation at any point in time. But it will be basically ignored until the maintenance window.

Reply:
Makes sense, thanks. I'll word accordingly.